src/BankID.php

```diff
@@ -75,6 +75,38 @@ public function authenticate($personalNumber, $ip)
         return new BankIDResponse(BankIDResponse::STATUS_PENDING, $httpResponseBody);
     }
 
+    /**
+     * Start a sign-order.
+     *
+     * @param $personalNumber
+     *
```

Comment:
Nitpick: Other docblocks in this file don't have an empty line between @param

Reply:
I initially copied the auth-method and there is an empty line, but i removed it from both.

Reply:
Oh, thanks!